platforms/ide/ide/src/integTest/groovy/org/gradle/plugins/ide/eclipse/EclipseMultiModuleIntegrationTest.groovy

```diff
@@ -126,9 +126,11 @@ project(':services:utilities') {
             }
 
             subprojects {
-                eclipse {
-                    project {
-                        name = rootProject.name + path.replace(':', '-')
+                if (project.name != 'nonEclipse') {
```

Comment:
We never applied `eclipse` to `:nonEclipse`, so instead we configured the root project's  `eclipse` container twice